storage/imx6ull-flash/flashdrv.c

```diff
@@ -187,11 +187,12 @@ static const flashdrv_command_t commands[flash_num_commands] = {
 };
 
 
-typedef struct _flashdrv_dma_t {
-	dma_t *last;
-	dma_t *first;
+typedef struct _flashdrv_dmaBuff_t {
```

Comment:
Ok, do we need this struct definition? I guess not, so it should be anonymous (typedef only)

Reply:
We need the struct definition because it's used in the header (see [link](https://github.com/phoenix-rtos/phoenix-rtos-devices/blob/b7ea02f1755c7f65656398cc351fc590ec8dd231/storage/imx6ull-flash/imx6ull-flashdrv.h#L20)).
The don't want the struct content to be visible, just the type.